### PR TITLE
feat: Updated sdk to include web app operations

### PR DESCRIPTION
**Pull request by yell66w**
Closes:
https://github.com/delta-storage/delta-storage-web-app/issues/220
https://github.com/delta-storage/delta-storage-web-app/issues/219
https://github.com/delta-storage/delta-storage-web-app/issues/217

### CODE REVIEW

File: src/web-app/file-access.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+  return result.data
+}
+
+export async function addFileAccess(
+  this: DeltaStorageSDK,
+  fileId: string,
+  cid: string,
+  email?: string,
+  password?: string
```
Comment (glendell03) [inline src/web-app/file-access.ts:33]: @yell66w pwede dito naka object? para pag gagamitin ganto

```tsx
addFileAccess(id, cid, {email})
addFileAccess(id, cid, {password})
```

File: src/web-app/file-access.ts [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+  return result.data
+}
+
+export async function deleteFileAccess(
+  this: DeltaStorageSDK,
+  fileId: string,
+  cid: string,
+  email: string,
+  deleteAll?: boolean
```
Comment (glendell03) [inline src/web-app/file-access.ts:58]: same here

File: src/web-app/settings.ts [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  })
+
+  return result.data
+}
+
+export async function updateSettings(
+  this: DeltaStorageSDK,
+  node?: string,
+  encryptionKey?: string,
+  isSecureMode?: boolean
```
Comment (glendell03) [inline src/web-app/settings.ts:18]: same

**Review by glendell03 — CHANGES_REQUESTED** [review]

File: src/web-app/file-access.ts (hunk 1 above)
Comment (yell66w) [reply src/web-app/file-access.ts:33]: onga noh. Sige change ko

**Review by glendell03 — APPROVED** [review]

**Pull request merged**